frontend/installation-ui/web/micro-ui-internals/packages/modules/pm/src/pages/employee/CreateProject.js

```
@@ -519,8 +523,20 @@ const CreateProject = () => {
 
   const onStepClick = (key) => {
-    if (key >= currentKey) return;
-    setCurrentKey(key + 1);
+    if (key + 1 >= currentKey) return;
+    switch (currentKey) {
+      case 2:
+        const geographyDetails = {
+          state: getFormData("state"),
+          districts: getFormData("districts"),
+          blocks: getFormData("blocks"),
+        }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)